third_party/mkl_dnn/mkldnn_acl.BUILD

```diff
@@ -111,6 +123,17 @@ _DNNL_RUNTIME_OMP = {
     "#cmakedefine01 BUILD_XEHPG": "#define BUILD_XEHPG 0",
     "#cmakedefine01 BUILD_XEHPC": "#define BUILD_XEHPC 0",
     "#cmakedefine01 BUILD_XEHP": "#define BUILD_XEHP 0",
+    "#cmakedefine01 BUILD_GROUP_NORMALIZATION": "#define BUILD_GROUP_NORMALIZATION 0",
+    "#cmakedefine01 BUILD_GEMM_KERNELS_ALL": "#define BUILD_GEMM_KERNELS_ALL 1",
+    "#cmakedefine01 BUILD_GEMM_KERNELS_NONE": "#define BUILD_GEMM_KERNELS_NONE 0",
+    "#cmakedefine01 BUILD_GEMM_SSE41": "#define BUILD_GEMM_SSE41 0",
+    "#cmakedefine01 BUILD_GEMM_AVX2": "#define BUILD_GEMM_AVX2 0",
+    "#cmakedefine01 BUILD_GEMM_AVX512": "#define BUILD_GEMM_AVX512 0",
+    "#cmakedefine DNNL_GPU_VENDOR": "#define DNNL_GPU_VENDOR INTEL",
+    "#cmakedefine DNNL_SYCL_GENERIC": "#undef DNNL_SYCL_GENERIC",
+    "#cmakedefine DNNL_DISABLE_GPU_REF_KERNELS": "#undef DNNL_DISABLE_GPU_REF_KERNELS",
+    "#cmakedefine01 BUILD_SDPA": "#define BUILD_SDPA 0",
+    "#cmakedefine01 BUILD_XE2": "#define BUILD_XE2 0",
```

Comment:
Let's avoid having to update same things in two places (`_DNNL_RUNTIME_THREADPOOL` and `_DNNL_RUNTIME_OMP`). Could you please refactor common options to `_CMAKE_COMMON_LIST` like in `mkldnn_v1.BUILD`?
https://github.com/tensorflow/tensorflow/blob/a944fd9a5abd82db10ea633c235228fe59f14199/third_party/mkl_dnn/mkldnn_v1.BUILD#L7-L73

Reply:
We do not actually support the OMP build of TF on aarch64 anymore. So I might go ahead and just delete the OMP list.

Reply:
Deleting the OMP list sounds good too.

Reply:
Have removed OMP builds for this platform

Reply:
@penpornk As a result of removing OMP support (the `mkl_aarch64` build is now an alias for `mkl_aarch64_threadpool`) I think [your patch](https://github.com/tensorflow/tensorflow/blob/eeb158cc4fa879f0a34e813cb3ed3b1a4557bb72/third_party/xla/third_party/compute_library/exclude_omp_scheduler.patch) is no longer necessary, and I have therefore deleted it. Could you please confirm if this is assumption is correct?